configs/default/CUST-SIRMIXALOT.config

```diff
@@ -0,0 +1,187 @@
+resource BARO_CS 1 A15
+resource FLASH_CS 1 C00
+resource OSD_CS 1 C14
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 B02
+
+# timer
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer B09 AF2
+# pin B09: TIM4 CH4 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature SOFTSERIAL
+feature TELEMETRY
+feature LED_STRIP
+feature OSD
+
+# beeper
+beeper -ON_USB
+
+# serial
+serial 2 2048 115200 57600 0 115200
+serial 3 1024 115200 57600 0 115200
+serial 4 64 115200 57600 0 115200
+serial 30 16384 115200 57600 0 115200
+
+# led
+led 0 7,7::A:2
+
+# color
+color 10 240,128,255
+
+# mode_color
+mode_color 6 1 2
+
+# aux
+aux 0 0 0 1700 2100 0 0
+aux 1 27 8 1300 2100 0 0
+aux 2 13 4 1300 2100 0 0
+aux 3 15 5 1300 2100 0 0
+aux 4 26 0 900 2100 0 0
+aux 5 33 2 1300 1700 0 0
+aux 6 34 3 1300 1700 0 0
+aux 7 36 6 1700 2100 0 0
+
+# vtxtable
+vtxtable bands 5
+vtxtable channels 8
+vtxtable band 1 BANDA    A FACTORY 5865 5845 5825 5805 5785 5765 5745 5725
+vtxtable band 2 BANDB    B FACTORY 5733 5752 5771 5790 5809 5828 5847 5866
+vtxtable band 3 BANDE    E FACTORY 5705 5685 5665 5645 5885 5905 5925 5945
+vtxtable band 4 BANDF    F FACTORY 5740 5760 5780 5800 5820 5840 5860 5880
+vtxtable band 5 RACE     R FACTORY 5658 5695 5732 5769 5806 5843 5880 5917
+vtxtable powerlevels 4
+vtxtable powervalues 25 200 500 800
+vtxtable powerlabels 25 200 500 800
+
+# master
+set acc_calibration = 24,-24,38
+set mag_hardware = NONE
+set baro_spi_device = 3
+set serialrx_provider = FPORT
+set serialrx_inverted = ON
+set serialrx_halfduplex = ON
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set dshot_bidir = ON
+set motor_pwm_protocol = DSHOT600
+set current_meter = NONE
+set battery_meter = ADC
+set vbat_scale = 80
+set ibata_scale = 250
+set beeper_inversion = ON
+set beeper_od = OFF
+set pid_process_denom = 1
+set ledstrip_visual_beeper = ON
+set osd_vbat_pos = 2080
+set osd_rssi_pos = 2048
+set osd_tim_1_pos = 2104
+set osd_tim_2_pos = 2136
+set osd_flymode_pos = 2074
+set osd_vtx_channel_pos = 480
+set osd_craft_name_pos = 2058
+set osd_home_dir_pos = 2144
+set osd_altitude_pos = 2112
+set osd_disarmed_pos = 2250
+set vtx_band = 1
+set vtx_channel = 3
+set vtx_power = 3
+set vtx_freq = 5825
+set max7456_spi_bus = 3
+set led_inversion = 3
+set dashboard_i2c_bus = 2
+set flash_spi_bus = 2
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180
+set gyro_1_align_yaw = 1800
+
+profile 0
+
+profile 1
+
+profile 2
+
+rateprofile 0
+
+rateprofile 1
+
+rateprofile 2
+
+rateprofile 3
+
+rateprofile 4
+
+rateprofile 5
+
+# 
```

Comment:
These should not be in here, as they are empty.

Reply:
Maybe I'll revise the instructions on how to make a config....